presto-verifier/src/test/java/com/facebook/presto/verifier/rewrite/TestQueryRewriter.java

```diff
@@ -294,6 +298,160 @@ public void testRewriteNonStorableStructuredTypes()
                         "    CAST(ROW (NULL) AS ROW(BIGINT))");
     }
 
+    @Test
+    public void testConstructMakeFunctionCallSubstituteMap()
+    {
+        assertEqualsDeep(FunctionCallRewriter.constructFunctionCallSubstituteMap("/func1(a,b)/func2(b)/"),
```

Comment:
This is a bit hard to read. The text is dense and there is a lot of repetition. Would it be possible to refactor?